test/login.tests.js

```diff
@@ -16,18 +16,24 @@ const filterRoute = (method, path) => {
     r.route && r.route.path === path && r.route.methods[method.toLowerCase()];
 };
 
-const fetchFromAuthCookie = (res, cookieName) => {
```

Comment:
Add a utility function to test, and Refactor other function to use it.

Referenced below in my new test cases.